apps/web/src/bridgeApi/bridgeApi.client.ts

```diff
@@ -24,8 +24,6 @@ export function buildBridgeHTTPClient(baseURL: string) {
     baseURL,
     headers: {
       'Content-Type': 'application/json',
-      // Required for localtunnel.it
-      'Bypass-Tunnel-Reminder': true,
```

Comment:
Not needed anymore.

Reply:
@rifont I kept this in place in case developers bring their tunnel via `localhost.it`.